devtools/quick_setup_script.sh

```diff
@@ -129,11 +129,11 @@ echo ~~~Setting up Contracts
 cd ../
 python -u devtools/contract_setup_script.py
 
-if [[ "mockData" != 'none' ]]; then
-    echo ~~~Creating mock data
-    cd ./app/server/utils
-    python -u mock_data.py ${testdata}
-fi
+#if [[ "mockData" != 'none' ]]; then
```

Comment:
I know it's broken, but should it be nuked in this PR?

Reply:
oh nope that was a mistake!